Gemfile

```diff
@@ -36,13 +36,16 @@ group :development, :test do
   gem 'brakeman', '~> 4.1', require: false
   gem 'byebug', '~> 9.1'
   gem 'dotenv-rails', '~> 2.2', '>= 2.2.1'
+  gem 'factory_bot_rails', '~> 4.8', '>= 4.8.2'
   gem 'rails-controller-testing', '~> 1.0', '>= 1.0.2'
   gem 'rspec-rails', '~> 3.7', '>= 3.7.2'
   gem 'rubocop', '~> 0.52.1', require: false
 end
 
 group :test do
-  gem 'factory_bot_rails', '~> 4.8', '>= 4.8.2'
+  gem 'capybara', '~> 2.17'
+  gem 'chromedriver-helper', '~> 1.1'
```

Comment:
[The chromedriver-helper gem](https://github.com/flavorjones/chromedriver-helper) will install the `chromedriver` binary to `~/.chromedriver-helper` the first time the test suite is invoked. I went this route based on [its inclusion in Rails' Gemfile](https://github.com/rails/rails/blob/master/Gemfile) rather than adding steps to the setup process to manually install the binary.